### PR TITLE
config: enable tichi for dm

### PR DESCRIPTION
**Pull request by hi-rustin**
part of https://github.com/ti-community-infra/configs/issues/173

### CODE REVIEW

**Comment by hi-rustin** [comment]
/hold

- [ ] Migration SIG check config 
- [ ] @sykp241095  Give the bot admin permission

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -591,6 +599,32 @@ ti-community-blunderbuss:
...
+    max_request_count: 2
+    exclude_reviewers:
+      # Bots
+      - ti-chi-bot
+      - ti-srebot
+      # Inactive reviewers for dm.
+      - amyangfei
+      - csuzhangxc
+      - glorv
+      - GMHDBJD
```
Comment (lance6716) [inline prow/config/external_plugins_config.yaml:614]:
```suggestion
```

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Comment by hi-rustin** [comment]
/lgtm cancel

Updated some configurations

@lance6716 PTAL ~

**Comment by hi-rustin** [comment]
/assign @Mini256 

PTAL~ Thanks!

**Review by Mini256 — APPROVED** [review]

**Pull request merged**